core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -335,30 +343,33 @@ object TopicCommand extends Logging {
                                         "broker_id_for_part2_replica1 : broker_id_for_part2_replica2 , ...")
                            .ofType(classOf[String])
     val reportUnderReplicatedPartitionsOpt = parser.accepts("under-replicated-partitions",
-                                                            "if set when describing topics, only show under replicated partitions")
+                                                            "If set when describing topics, only show under replicated partitions.")
     val reportUnavailablePartitionsOpt = parser.accepts("unavailable-partitions",
-                                                            "if set when describing topics, only show partitions whose leader is not available")
+                                                            "If set when describing topics, only show partitions whose leader is not available.")
     val topicsWithOverridesOpt = parser.accepts("topics-with-overrides",
-                                                "if set when describing topics, only show topics that have overridden configs")
+                                                "If set when describing topics, only show topics that have overridden configs.")
     val ifExistsOpt = parser.accepts("if-exists",
-                                     "if set when altering or deleting topics, the action will only execute if the topic exists")
+                                     "If set when altering or deleting topics, the action will only execute if the topic exists.")
     val ifNotExistsOpt = parser.accepts("if-not-exists",
-                                        "if set when creating topics, the action will only execute if the topic does not already exist")
+                                        "If set when creating topics, the action will only execute if the topic does not already exist.")
 
-    val disableRackAware = parser.accepts("disable-rack-aware", "Disable rack aware replica assignment")
+    val disableRackAware = parser.accepts("disable-rack-aware", "Disable rack aware replica assignment.")
 
-    val forceOpt = parser.accepts("force", "Suppress console prompts")
+    val forceOpt = parser.accepts("force", "Suppress console prompts.")
 
-    val options = parser.parse(args : _*)
+    val commandDef: String = "Create, delete, describe, or change a topic."
+    
+    if(args.length == 0)
```

Comment:
nit: missing whitespace after `if`. Seems to be recurring in most of these files.